mgroovyank/Day 89/Longest valid Parentheses.cpp

```diff
@@ -1,37 +1,28 @@
-// https://www.interviewbit.com/problems/longest-valid-parentheses/
```

Comment:
You are not allowed to make changes to the program of others. Please make sure that you commit only your changes